### ISSUE TITLE
Description text not validated

### ISSUE THREAD

**Issue by rpkilby** [terrencepreilly/darglint#91]
Per google's style guide, a blank line is required between the summary and the remainder  of the description. The below validates without error.

```python
def foo(bar):
    """Summary.
    Long description.

    Args:
        bar: arrrggg
    """
    pass
```

edit:
It looks like the summary and description aren't actually validated at all. The following pass:

```python
# blank lines before summary
def foo(bar):
    """


    Summary.
    Long description.

    Args:
        bar: arrrggg
    """
    pass


# no summary/description
def foo(bar):
    """
    Args:
        bar: arrrggg
    """
    pass
```

**Comment by rpkilby** [comment]
Related, I'm wondering if the summary is required to be on the same line as the opening quotes. The examples all show this, however the style guide does not explicitly state that it's required:

> A docstring should be organized as a summary line (one physical line) terminated by a period, question mark, or exclamation point, followed by a blank line, followed by the rest of the docstring starting at the same cursor position as the first quote of the first line.

Should the following be valid/invalid?

```python
def foo(bar):
    """
    Summary.

    Long descrtiption.

    Args:
        bar: arrrggg
    """

```

**Comment by terrencepreilly** [comment]
>  It looks like the summary and description aren't actually validated at all.

Correct.  Summary and long descriptions are validated by `pydocstyle`.  I'm not sure that that's the best strategy, and we could always change it.

> Related, I'm wondering if the summary is required to be on the same line as the opening quotes. The examples all show this, however the style guide does not explicitly state that it's required

I've wondered about it, too.  Since there's no clear guidance, darglint doesn't attempt to enforce a particular style.  There are a significant number of larger projects which put the summary on a newline.  We could always add a style error for this case, and have it disabled by default.

Overall, I'm pretty open to however people want to handle this.  Since these are pretty minor (and likely to be very annoying) error to raise, I would be willing to add disabled-by-default options for this.

**Comment by rpkilby** [comment]
> Correct. Summary and long descriptions are validated by `pydocstyle`. 

That makes sense to me. What I think is confusing here is that the user needs to know to call `pydocstyle` separately. My naive expectation would be that calling `darglint` would result in the full docstring being validated. Maybe the readme should mention `pydocstyle`?

> I'm not sure that that's the best strategy, and we could always change it.

In general, it makes sense to me that `darglint` delegates description validation to `pydocstyle`. However, I'm not sure if `darglint` should be responsible for calling `pydocstyle`, or if it's the responsibility of the user. This is also complicated by other flake8 plugins like flake8-docstrings. 

Maybe darglint should have the option to call into `pydocstyle`, and this behavior can be enabled or disabled? 

> I've wondered about it, too. Since there's no clear guidance, darglint doesn't attempt to enforce a particular style.

Yeah - I was always under the assumption that same line summary was a requirement, but I think that might just be an implicit convention? Either way, I'd agree that the best thing to do here is rely on `pydocstyle`.

**Comment by terrencepreilly** [comment]
> What I think is confusing here is that the user needs to know to call pydocstyle separately. My naive expectation would be that calling darglint would result in the full docstring being validated. Maybe the readme should mention pydocstyle?

That's a good point.  I'll add a note of that in the readme.

I think it's probably a good idea to allow this check in the future, but have it disabled by default.  I'm a little more concerned with improving speed and stability at the moment, so I've added more aggressive style checking to the roadmap in the readme.
